src/test/resources/et1VettingTest1.json

```diff
@@ -10,10 +10,21 @@
         "Country": "United Kingdom"
       }
     },
+    "claimantIndType": {
+      "claimant_last_name": "Johnson",
+      "claimant_first_names": "Doris"
+    },
```

Comment:
Is it actually necessary to have three json files for different tests? I think it should be possible to use the exsiting case data builder? (correct me if I am wrong)